pyomo/dataportal/tests/test_dataportal.py

```diff
@@ -65,7 +66,7 @@ def test_read_set(self):
             td.read()
             td.close()
             self.assertEqual( td._info, ['set', 'X', ':=', ('A1', 2.0, 3.0, 4.0), ('A5', 6.0, 7.0, 8.0), ('A9', 10.0, 11.0, 12.0), ('A13', 14.0, 15.0, 16.0)])
-        except pyutilib.common.ApplicationError:
+        except ApplicationError:
```

Comment:
Do we know who is raising this `ApplicationError`?  Why are we allowing these exceptions to be "implicitly passed tests"?

Reply:
It looks like the DataManagerFactory functions (open, read, close) are raising this error - at least, that's what the code seems to imply. I'm not sure why that would raise an `ApplicationError`, though.